pinot-core/src/main/java/com/linkedin/pinot/core/operator/aggregation/function/MinMaxRangeAggregationFunction.java

```diff
@@ -162,7 +168,7 @@ public MinMaxRangePair merge(@Nonnull MinMaxRangePair intermediateResult1,
   @Nonnull
   @Override
   public FieldSpec.DataType getIntermediateResultDataType() {
-    return FieldSpec.DataType.DOUBLE;
+    return FieldSpec.DataType.OBJECT;
```

Comment:
This is a typo in the new added code (in clean up step 1), and the unit test caught it.